miasm2/expression/expression.py

```diff
@@ -760,13 +761,20 @@ def __init__(self, op, *args):
                           'int_16_to_double', 'int_32_to_double',
                           'int_64_to_double', 'int_80_to_double']:
             sz = 64
-        elif self._op in ['double_to_mem_16', 'double_to_int_16', 'double_trunc_to_int_16']:
+        elif self._op in ['double_to_mem_16', 'double_to_int_16',
```

Comment:
For `double_to_mem_`, `double_to_int_`, `double_trunc_to_int_` it seems that the size is embedded in the `ExprOp` name.
Using a ```if ... startswith``` and getting the size at the end  may allow to automatically supporting every size for these instructions (if this is a wanted behavior), avoiding copying the same code for each new size to support.